<?xml version="1.0"?>
<launch>
  <arg name="output_file" default="" doc="yaml file to store benchmarking data" />
  <arg name="visualize" default="false" doc="turns on and off visualization with rviz (bool type)" />
  <arg name="debug" default="false" doc="run in gdb" />

  <arg name="name" default="Collision Check Default" />

  <include file="$(dirname)/collision_check.launch.inc" pass_all_args="true">
    <arg name="config_file" value="$(dirname)/../config/collision_check.yaml" />
  </include>
</launch>
